config/initializers/sidekiq.rb

```diff
@@ -31,6 +31,11 @@
         "cron" => "0 19 * * *",
         "args" => ENV["LOG_BUCKET_NAME"],
       },
+      "daily_ingest_of_cloudfront_logs" => {
+        "class" => "IngestCloudfrontLogWorker",
+        "cron" => "0 20 * * *",
+        "args" => ENV["LOG_BUCKET_NAME"],
```

Comment:
this env var should probably be named differently to the one for clf logs
Also i can see us having more than one bucket in the future.

Reply:
👍  Where are these set? If I rename them will it mess anything up?

Reply:
From our point of view nowhere currently. So changing them is fine.